### ISSUE TITLE
Logo alt text should describe back to home page

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#10245]
**What**
"It's just saying Wellcome collection, but because this acts as a home link, ideally the description should be clearer.

It should say something like Wellcome Collection homepage link because some screen reader users elements if they get lost may look for something that's specifically says home or homepage or go back to home."

**Where**
Logo alt text

**TODO**

- [x] Amend alt text to 'Wellcome Collection home page'

**Background**

From the Hassell Inclusion accessibility audit August 2023.

Dovetail: https://proficient-wallaby-dpwd.dovetailapp.com/insights/Logo-alt-text-should-be-more-descriptive-5xjBPgXJZ8d1uGtdrQBCw2

Miro board: https://miro.com/app/board/uXjVMjsL5YE=/
